dust_attenuation/shapes.py

```diff
@@ -0,0 +1,391 @@
+# -*- coding: utf-8 -*-
+
+import numpy as np
+import astropy.units as u
+
+from .baseclasses import BaseAttAvModel
+from .helpers import _test_valid_x_range
+
+from .averages import C00, Leitherer02
+from astropy.modeling import Parameter, InputParameterError
+
+__all__ = ['Noll09']
```

Comment:
Given our naming convention, should be 'N09'.  Unless you think we should change the convention.  :-)

http://dust-attenuation.readthedocs.io/en/latest/dust_attenuation/references.html#naming-convention

Reply:
The convention is perfectly fine :) I just did not know it... I'll make the changes